WooCommerce/WooCommerceTests/Tools/MockOrders.swift

```diff
@@ -131,6 +141,19 @@ final class MockOrders {
         taxes: [])]
     }
 
+    func makeOrderTaxLine(totalShippingTax: String,
+                          totalTax: String) -> OrderTaxLine {
+        .init(taxID: 123,
+              rateCode: "",
+              rateID: 1,
+              label: "State",
+              isCompoundTaxRate: false,
+              totalTax: totalTax,
+              totalShippingTax: totalShippingTax,
+              ratePercent: 5.5,
+              attributes: [])
```

Comment:
Since `OrderTaxLine` conforms to `GeneratedFakable` and `GeneratedCopiable`, this can be simplified as: 
```suggestion
        OrderTaxLine.fake().copy(totalTax: totalTax, totalShippingTax: totalShippingTax)
```
For more information about Copiable and Fakable, please check out the docs [here](https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/copiable.md) and [here](https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/fakeable.md).